src/lib/core/a11y/list-key-manager.ts

```diff
@@ -63,6 +64,9 @@ export class ListKeyManager<T extends CanDisable> {
       case END:
         this.setLastItemActive();
         break;
+      case ESCAPE:
```

Comment:
I'm not sure the escape handling really belongs here; I think the problem is that while it's called `ListKeyManager`, it's really more like `ListItemMovementKeyManager`. The fact that there's both `tab` and `escape` as observables with no behavior seems like a code smell; perhaps the components should handle these things themselves. 

@kara thoughts?